core/templates/dev/head/domain/exploration/ExplorationObjectFactory.js

```diff
@@ -117,6 +117,10 @@ oppia.factory('ExplorationObjectFactory', [
       return this.getState(this.initStateName);
     };
 
+    Exploration.prototype.getContentHtml = function(stateName) {
```

Comment:
I just realized that we should probably call this something like Exploration.prototype.getUninterpolatedContentHtml, since it may contain parameters. (This is as opposed to the HTML that is eventually displayed as the content.)